app/dashboard/agenda/page.tsx

```
@@ -1,7 +1,148 @@
+            <div>
+              <h1 className="text-xl font-semibold text-blue-800">Fernando de Souza</h1>
+              <p className="text-sm text-gray-500">Matrícula: 43-0295810457</p>
+              <p className="text-sm text-gray-500">Categoria: AB</p>
+              <p className="text-sm text-gray-500">Estágio: Aulas Práticas</p>
+            </div>
+          </div>
+
+          {/* Botões */}
+          <div className="space-x-2">
+            <button className="bg-blue-600 text-white py-1 px-4 rounded hover:bg-blue-700">
+              Agendar
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)